### ISSUE TITLE
Wrong aws_autoscaling_group calculation for spot instances

### ISSUE THREAD

**Issue by karnauskas** [infracost/infracost#104]
I have `aws_launch_template` which is using spot instances / spot bidding with autoscaling group, however infracost is showing those instances as `on-demand`:

```
  aws_autoscaling_group.spot_asg
  ├─ Compute (on-demand, t3a.nano) (aws_launch_template.xxx)          730  hours      0.0053       0.0053        3.8690

```

**Comment by aliscott** [comment]
@karnauskas thanks. If it's possible can you add the terraform code that produced this?

**Comment by aliscott** [comment]
Closing since this may be fixed in #142. Please re-open if this can be reproduced.
